homeassistant/components/homeassistant_alerts/__init__.py

```diff
@@ -197,9 +176,10 @@ async def _async_update_data(self) -> dict[str, IntegrationAlert]:
                     continue
 
                 integration_alert = IntegrationAlert(
+                    alert_id=alert["id"],
                     integration=integration["package"],
                     filename=alert["filename"],
-                    date_updated=alert.get("date_updated"),
+                    date_updated=alert.get("updated"),
```

Comment:
Technically an unrelated bugfix, `date_updated` never existed, it was always `updated`.